tatted_dream/server/email/send_email.py

```diff
@@ -0,0 +1,23 @@
+from tatted_dream.server.google.Google import Create_Service
+import base64
+import os
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+CLIENT_SECRET_FILE = 'D:\Personal-Projects\Tatted-Dream\client_secret.json'
+API_NAME = 'gmail'
+API_VERSION = 'v1'
+SCOPES = ['https://mail.google.com/']
+email = os.environ["email"]
```

Comment:
trying to understand this part of the program. where are the modules being imported from? also testing my first pull request review